.circleci/config.yml

```diff
@@ -0,0 +1,145 @@
+aliases:
+  - &restore_sbt_cache
+    restore_cache:
+      key: sbt-{{ checksum "/tmp/scala_version" }}-cache
+
+  - &save_sbt_cache
+    save_cache:
+      key: sbt-{{ checksum "/tmp/scala_version" }}-cache-{{ epoch }}
+      paths:
+        - "~/.ivy2/cache"
+        - "~/.sbt"
+        - "~/cache/coursier"
+
+  - &attach_workspace
+    attach_workspace:
+      at: "~"
+
+  - &persist_to_workspace
+    persist_to_workspace:
```

Comment:
We want to persist the results of `test:compile` to the current build so that we can avoid pulling dependencies again when running `test`.